File: api/DropShot.Application/Drops/Models/DropItemDto.cs
using DropShot.Application.Common.AutoMapper;
using DropShot.Domain.Entities;
using DropShot.Domain.Enums;

namespace DropShot.Application.Drops.Models;

public class DropItemDto
{
    public int DropItemId { get; set; }
    public int VariantId { get; set; }
    public int ProductId { get; set; }
    public string ProductName { get; set; }
    public ClothesUnitOfMeasure UnitOfSize { get; set; }
    public int Size { get; set; }
    public DateTime? ReservationEndDateTime { get; set; }
}

File: web/src/api/models/Drops/DropItemDto.ts
import { ClothesUnitOfMeasure } from '../Products/ClothesUnitOfMeasure';

export interface DropItemDto {
  dropItemId: number;
  variantId: number;
  productId: number;
  productName: string;
  unitOfSize: ClothesUnitOfMeasure;
  size: number;
}
